suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -92,12 +96,15 @@ export const XpubScanScreen = ({
 
     const onXpubFormSubmit = handleSubmit(goToAccountImportScreen);
 
-    const handleXpubResult = (xpubAddress?: string) => {
-        if (xpubAddress) {
-            setValue('xpubAddress', xpubAddress);
-            onXpubFormSubmit();
-        }
-    };
+    const handleXpubResult = useCallback(
```

Comment:
So whole problem was just missing useCallback?

Reply:
I would say "the main" problem.

As you can see, there is `<Camera onResult={handleXpubResult} />` component for camera scanning. `handleXpubResult` sets the value for input field: `setValue('xpubAddress', xpubAddress);` and if you set the value, the whole form is updated and `handleXpubResult` passed as `onResult` prop to the component `Camera` is changed.

`onResult` is one of the array dependencies for useEffect in `Camera`... so `handleXpubResult` was called again and again and again...